tls.md

```diff
@@ -1,5 +1,7 @@
 # TLS
 
+Certificate management within the openstack-operator is per default done using [cert-manager](https://cert-manager.io/docs/). For each component the openstack-operator creates one or multiple certificates and pass the name/names of the k8s secret, which contains the certificate, key and ca certificate, to the service component. Per default cert-manager does not delete k8s secrets, holding the certificate information, when the cert-manager certificate resource gets deleted. Consult the OpenShift documentation on [Deleting a TLS secret automatically upon Certificate removal](https://docs.openshift.com/container-platform/4.14/security/cert_manager_operator/cert-manager-customizing-api-fields.html#cert-manager-override-flag-controller_cert-manager-customizing-api-fields).
```

Comment:
nit: "by default" is more common than "per default"

Reply:
done